central/processindicator/datastore/datastore_impl_test.go

```diff
@@ -405,24 +405,28 @@ func (suite *IndicatorDataStoreTestSuite) TestAllowsGet() {
 	protoassert.Equal(suite.T(), testIndicator, indicator)
 }
 
+// TODO: replace this mocked test with a Postgres integration one
```

Comment:
How many of these TODOs are actually covered in the SAC test?  Often when I've encountered such cases the SAC test actually covered most cases making these duplicative and subject for removal.

Reply:
I did check by removing these `Enforce` tests and extending the SAC tests and adding the missing related SAC tests in #17331   .
It seems to cover both code paths where SAC checks accept and reject the action. I'll remove the `TestEnforceXXX` functions from the mocked tests along with the comments.